home/make-user.js

```diff
@@ -0,0 +1,22 @@
+// build a simple JS user out of the weird formdata object
+function makeUserData(formData) {
+    const user = {
+        name: formData.get('name'),
+        race: formData.get('race'),
+        hp: 35,
+        gold: 0,
+        completed: {}
+    };
+
+    return user;
+}
+
+export const newMakeUser = (formData) => ({
+    name: formData.get('name'),
+    race: formData.get('race'),
+    hp: 35,
+    gold: 0,
+    completed: {}
+});
```

Comment:
This function does the same as the function above and it doesn't seem to be imported anywhere so you can get rid of it.